bot/cogs/bot.py

```diff
@@ -60,6 +62,17 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
+        if msg.content.count("\n") >= 3:  # more than three lines
+            try:
+                tree = ast.parse(msg.content)
+                # tries to parse the code into a valid ast if it throws a SyntaxError it is no valid python
```

Comment:
May I suggest `# Attempts to parse the message into an AST node.`
`# Invalid Python code will raise a SyntaxError.`